google/cloud/exceptions.py

```diff
@@ -41,7 +43,10 @@ def __init__(self, message, errors=()):
         self._errors = errors
 
     def __str__(self):
-        return '%d %s' % (self.code, self.message)
+        result = u'%d %s' % (self.code, self.message)
+        if six.PY2:  # pragma: NO COVER  Python2
```

Comment:
Well `tox -e cover` runs py2....so I'm not sure why it's not entering that branch?